CONTRIBUTING.md

```diff
@@ -121,3 +121,4 @@ Existing maintainers can resign, or be subject to a vote for removal at the behe
 * [legleux](https://github.com/legleux) (Ripple)
 * [undertome](https://github.com/undertome) (Ripple)
 * [godexsoft](https://github.com/godexsoft) (Ripple)
+* [officialfrancismendoza](https://github.com/officialfrancismendoza) (Ripple)
```

Comment:
Not sure how this change got into this pr but it should not be (it was in #285 already)